src/main/java/org/apache/mesos/hadoop/Utils.java

```diff
@@ -34,4 +40,96 @@ public static ByteString confToBytes(Configuration conf) throws IOException {
     byte[] bytes = baos.toByteArray();
     return ByteString.copyFrom(bytes);
   }
+
+  public static CommandInfo.ContainerInfo buildContainerInfo(Configuration conf) {
+    String containerImage = conf.get("mapred.mesos.container.image");
+    String[] containerOptions = conf.getStrings("mapred.mesos.container.options");
+
+    CommandInfo.ContainerInfo.Builder containerInfo =
+        CommandInfo.ContainerInfo.newBuilder();
+
+    if (containerImage != null) {
+      containerInfo.setImage(containerImage);
+    }
+
+    if (containerOptions != null) {
+      for (int i = 0; i < containerOptions.length; i++) {
+        containerInfo.addOptions(containerOptions[i]);
+      }
+    }
+
+    return containerInfo.build();
+  }
+
+  public static ContainerInfo buildDockerContainerInfo(Configuration conf) {
+    ContainerInfo.Builder containerInfoBuilder = ContainerInfo.newBuilder();
+    ContainerInfo.DockerInfo.Builder dockerInfoBuilder = ContainerInfo.DockerInfo.newBuilder();
```

Comment:
Again, I'd be tempted to import `ContainerInfo`, and potentially `ContainerInfo.DockerInfo` for brevity and clarity.